projects/systelab-components/ng-package.json

```diff
@@ -1,12 +1,25 @@
 {
   "$schema": "../../node_modules/ng-packagr/ng-package.schema.json",
-  "dest": "../../dist/systelab-components",
   "assets": [
     "./README.md",
     "./icons/*.*",
     "./sass/**/*.scss"
   ],
+  "dest": "../../dist/systelab-components",
   "lib": {
-    "entryFile": "src/public-api.ts"
+    "entryFile": "src/public-api.ts",
+    "umdModuleIds": {
+      "ag-grid-angular": "ag-grid-angular",
+      "angular-split": "angular-split",
+      "date-fns": "date-fns",
+      "mobile-drag-drop": "mobile-drag-drop",
+      "primeng/api": "primeng/api",
```

Comment:
PrimeNG has not released yet the migration guide to v12 so it cannot be updated.
https://github.com/primefaces/primeng/wiki/Migration-Guide

Reply:
ok, it has not been upgraded